package.json

```
@@ -68,30 +67,30 @@
 		"test": "npm run test:mocha && npm run test:jest && npm run test:realsvc && npm run test:stress",
 		"test:bail": "npm run test:mocha:bail && npm run test:jest:bail",
 		"test:benchmark:copyresults": "copyfiles --exclude \"**/node_modules/**\" \"**/benchmarkOutput/**\" benchmarkOutput",
-		"test:benchmark:report": "lerna run test:benchmark:report --stream --no-bail --no-sort --reporterOptions reportDir=./benchmarkOutput",
+		"test:benchmark:report": "pnpm run -r --no-sort --stream --no-bail test:benchmark:report --reporterOptions reportDir=./benchmarkOutput",
 		"test:copyresults": "copyfiles --exclude \"**/node_modules/**\" \"**/nyc/**\" nyc",
 		"test:coverage": "c8 npm run test",
 		"test:fromroot": "mocha \"packages/**/dist/test/**/*.spec.js\" --exit",
-		"test:jest": "assign-test-ports && lerna run test:jest --concurrency 4 --stream --no-bail --no-sort -- --color",
-		"test:jest:bail": "assign-test-ports && lerna run test:jest --concurrency 4 --stream",
-		"test:jest:report": "assign-test-ports && lerna run test:jest --concurrency 4 --stream --no-bail --no-sort -- --ci --reporters=default --reporters=jest-junit",
+		"test:jest": "assign-test-ports && pnpm run -r --no-sort --stream --no-bail test:jest --color",
+		"test:jest:bail": "assign-test-ports && pnpm run -r --no-sort --stream test:jest",
+		"test:jest:report": "assign-test-ports && pnpm run -r --no-sort --stream --no-bail test:jest --ci --reporters=default --reporters=jest-junit",
 		"test:memory-profiling:copyresults": "copyfiles --exclude \"**/node_modules/**\" \"**/memoryProfilingOutput/**\" memoryProfilingOutput",
-		"test:memory-profiling:report": "lerna run test:memory-profiling:report --stream --no-bail --no-sort -- -- --reporterOptions reportDir=./memoryProfilingOutput",
-		"test:mocha": "lerna run test:mocha --stream --no-bail --no-sort -- --color",
-		"test:mocha:bail": "lerna run test:mocha --stream",
-		"test:mocha:report": "lerna run test:mocha:multireport --stream --no-bail --no-sort -- -- --timeout 4s",
-		"test:realsvc": "lerna run test:realsvc --stream --no-bail --no-sort",
-		"test:stress": "lerna run test:stress --stream --no-bail --no-sort",
-		"tsc": "lerna run tsc --stream",
-		"tsc:fast": "fluid-build  --root . -s tsc",
+		"test:memory-profiling:report": "pnpm run -r --no-sort --stream --no-bail test:memory-profiling:report -- --reporterOptions reportDir=./memoryProfilingOutput",
+		"test:mocha": "pnpm run -r --no-sort --stream --no-bail test:mocha --color",
+		"test:mocha:bail": "pnpm run -r --no-sort --stream test:mocha",
+		"test:mocha:report": "pnpm run -r --no-sort --stream test:mocha:multireport -- --timeout 4s",
```

I think we need this to keep behavior. Or was it removed deliberately?
```suggestion
		"test:mocha:report": "pnpm run -r --no-sort --stream --no-bail test:mocha:multireport -- --timeout 4s",
```

---

thats a bug. I'm going to double check I didn't mess up any other no-bail